bot/seasons/valentines/movie_generator.py

```diff
@@ -0,0 +1,61 @@
+import discord
+import logging
+from discord.ext import commands
+import random
+import json
+from urllib import parse, request
```

Comment:
`urllib` is blocking and can't be used with `discord.py`. [`aiohttp`](https://aiohttp.readthedocs.io/en/stable/) should be used instead, and is already installed for this project.

See the [`scarymovie` cog](https://github.com/python-discord/seasonalbot/blob/master/bot/seasons/halloween/scarymovie.py) for another example.

Reply:
Can i only use parse from urlib ?

Reply:
Sure